src/utils/input-validation.ts

```diff
@@ -0,0 +1,8 @@
+import { User } from "../models/user-model";
+
+export function isEmptyInput(user: User) {
+  if (user.email === '' || user.password === '') {
```

Comment:
You can use this format in the condition: `if (!user.email || !user.password)`

Reply:
If you have to return a boolean value based on a condition, you can return the result of the condition directly. You don't need to set the `return true` and `return false`, it is redundant in this case.
The result of the (!user.email || !user.password) is a boolean value.